src/select.tsx

```diff
@@ -357,12 +356,20 @@ const Select: React.FC<SelectProps> = ({
                             // Override default browser opacity when disabled. This opacity also affects the label.
                             // Without this fix, the label is invisible when disabled
                             opacity: 1,
+                            // Use transparent color for the empty option (show below) to avoid showing it when the menu is closed
+                            color: selectedValue ? undefined : 'transparent',
                         }}
                     >
-                        {options.every(({value}) => !!value) && (
-                            // if no "empty" option exists, insert a dummy empty option
-                            // this is needed to allow a native select with no selected option
-                            <option value="" style={{display: 'none'}} />
+                        {!selectedValue && options.every(({value}) => !!value) && (
+                            // If no "empty" option exists, insert a dummy empty option. Once an option is selected,
+                            // this empty option is removed. This is needed to allow a native select without a selected
+                            // default option.
+                            // Chrome doesn't show this option when the select menu is open (because of display: none),
+                            // but Safari does. So we use the select's label for this option, otherwise it would be shown
+                            // as an empty option, which is weirder.
+                            <option value="" style={{display: 'none'}} aria-label="">
```

Comment:
the empty `aria-label` is needed to avoid screen reader to read the select label twice